.github/workflows/build-test.yml

```diff
@@ -0,0 +1,43 @@
+name: Snyk Vulnerability Scan
```

Comment:
See the same feedback that is on the UI PR and apply it to this one

Reply:
change the title